### PR TITLE
Added subcommand to simply ask whether a given printer is supported

### PR DESCRIPTION
**Pull request by Bhavna2020**
This fixes issue #158.

### CODE REVIEW

File: pappl/mainloop-subcommands.c [hunk 1]
```diff
@@ -122,6 +122,59 @@ _papplMainloopAddPrinter(
...
+						
+  device_id = cupsGetOption("device-id", num_options, options);
+  
+  if (!device_id)
+  {
+   	fprintf(stderr, "%s: Missing '-o device-id=DEVICE-ID'.\n", base_name);
+    return(1);
+  }
+  
+  if (!system_cb)
```
Comment (michaelrsweet) [inline pappl/mainloop-subcommands.c:151]: The system callback doesn't need to be set - look at the code for the drivers sub-command...

File: pappl/mainloop.c [hunk 2]
```diff
@@ -84,6 +84,7 @@ papplMainloop(
   {					// List of standard sub-commands
     "add",
     "autoadd",
+    "is-supporting",
```
Comment (michaelrsweet) [inline pappl/mainloop.c:87]: I would prefer to see this become part of the "drivers" sub-command.

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
Please add support for the "device-id" option to the "drivers" sub-command.

**Comment by Bhavna2020** [comment]
OK.

**Pull request closed without merging**